Shopify/cli
# Remove legacy customer account extension code from CLI

### WHY are these changes introduced?

1. Remove categories from payload.
2. Remove legacy customer account ui extension type
3. Remove legacy customer account extension migrator

### How to test your changes?

See [this PR](https://github.com/Shopify/cli/pull/2837) for the general approach to 🎩. Ensure that you can startup the dev server and that the extensions render properly

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
